common/rya.api/src/main/java/org/apache/rya/api/instance/RyaDetailsToConfiguration.java

```diff
@@ -51,17 +51,29 @@ public static void addRyaDetailsToConfiguration(final RyaDetails details, final
 
         checkAndSet(conf, ConfigurationFields.USE_ENTITY, details.getEntityCentricIndexDetails().isEnabled());
         checkAndSet(conf, ConfigurationFields.USE_FREETEXT, details.getFreeTextIndexDetails().isEnabled());
-      //RYA-215        checkAndSet(conf, ConfigurationFields.USE_GEO, details.getGeoIndexDetails().isEnabled());
+        //RYA-215        checkAndSet(conf, ConfigurationFields.USE_GEO, details.getGeoIndexDetails().isEnabled());
         checkAndSet(conf, ConfigurationFields.USE_TEMPORAL, details.getTemporalIndexDetails().isEnabled());
         final PCJIndexDetails pcjDetails = details.getPCJIndexDetails();
-        if (pcjDetails.isEnabled() && pcjDetails.getFluoDetails().isPresent()) {
-            checkAndSet(conf, ConfigurationFields.USE_PCJ_UPDATER, true);
-            conf.set(ConfigurationFields.FLUO_APP_NAME, pcjDetails.getFluoDetails().get().getUpdateAppName());
-            conf.set(ConfigurationFields.PCJ_UPDATER_TYPE, "FLUO");
-            conf.set(ConfigurationFields.PCJ_STORAGE_TYPE, "ACCUMULO");
+        
+        if(conf.getBoolean(ConfigurationFields.USE_MONGO, false) ) {
+        	if(pcjDetails.isEnabled()) {
+        		conf.set(ConfigurationFields.PCJ_STORAGE_TYPE, "MONGO");
```

Comment:
Can "MONGO" be replaced with PrecomputedJoinStorageType.MONGO.toString()?  Or is the dependency for that class not in rya.api?

Reply:
there's no scope to it from rya.api